CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ For older changes see the [archived Singularity change log](https://github.com/a
   instead of underlay layout.
 - Add helpful error message for build `--remote` option.
 
+- Add more helpful error message when no library endpoint found.
```

Comment:
There shouldn't be a blank between these.

More importantly, because these two messages are going to the release-1.1 branch, even though they aren't bug fixes our convention is to put them in the Bug fixes section below.  That's so we'll remember to remove them in the main branch after they have been released in the release branch.  So please move these two messages down to that section.